CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Supported Release 4.0.1
+### Summary
+This is a small bugfix release that makes `mysql_install_db` optional and fixes some regular expression issues.
+
+#### Bugfixes
+- (MODULES-5528) Fixes the `mysql_install_db` command so that it is optional
```

Comment:
we're trying to do ticket numbers at the end of the line with links
```markdown
([MODULES-5528](https://tickets.puppet.com/browse/MODULES-5528))
([MODULES-5602](https://tickets.puppet.com/browse/MODULES-5602))
```

otherwise, looks great!